tvapp/src/main/java/com/sanaa/tvapp/TvApp.kt

```diff
@@ -0,0 +1,8 @@
+package com.sanaa.tvapp
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class TvApp: Application() {
+}
```

Comment:
Application class for the TV module currently missing onCreate() for initialization
```suggestion
@HiltAndroidApp
class TVApp : Application() {

    override fun onCreate() {
        super.onCreate()
    }
}
```